src/index.html

```diff
@@ -11,8 +14,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__background"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+      />
+
+      <div class="card__text">
+        <p class="card__text card__text--name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__text card__text--description">Product code: 195434</p>
+      </div>
+    </div>
+
+    <div class="stars">
```

Comment:
Add the modifier class
```suggestion
    <div class="stars stars--4">
```